### PR TITLE
Adds spare trader IDs

### PR DESCRIPTION
**Pull request by beesil**
[qol]
Adds three spare trader IDs to the tradevend, purchasable for 100 credits each. These IDs have access to the traderpost, and can be set to a name of your choice ONCE. 

The intention of this is to give traders a way to give mushroom men they grow full access to the outpost, as there is currently no way to make more trader IDs. The tradervend is locked to trader access, so spare trader IDs cannot be used by sec/miners to raid the outpost. 

:cl:
 * rscadd: Adds three spare trader IDs to the tradevend, purchasable for 100 credits each.

### CODE REVIEW

**Comment by Irradiation** [comment]
Refresh my memory a bit, can you hack Tradervends to make them spill out their contents?

**Comment by beesil** [comment]
@Irradiation 
> Refresh my memory a bit, can you hack Tradervends to make them spill out their contents?

Nope, they're unhackable.

**Comment by MadmanMartian** [comment]
Isn't there spare IDs in the safe?

**Comment by Dacendeth** [comment]
@MadmanMartian There's sometimes a captains spare id in the big safes, perhaps that's what you're thinking of?

File: code/game/objects/items/trader.dm [hunk 1]
```diff
@@ -726,3 +726,21 @@
 		return TRUE
 	else
 		return FALSE
+
+/obj/item/weapon/card/id/vox/extra
+	name = "Spare trader ID"
+	desc = "A worn looking ID with access to the tradepost, able to be set once for aspiring traders."
+	assignment = "Trader"
+	var canSet = TRUE
```
Comment (MadmanMartian) [inline code/game/objects/items/trader.dm]: var/canSet

**Comment by beesil** [comment]
@MadmanMartian 
> Isn't there spare IDs in the safe?

The outpost safe spawns with a 1000 credit chip, gold, silver, and diamonds in it.

File: code/game/objects/items/trader.dm (hunk 1 above)
Comment (beesil) [reply code/game/objects/items/trader.dm]: fixed

**Pull request merged**